File: scala/src/main/scala/io/digitalstream/library/Tuples.scala
/**
  * Created by sebby on 9/25/16.
  */
package io.digitalstream.library

object Tuples {

  def run(): Unit = {
    val tuple = (1, "hello")
    val tuple2: Tuple2[Int, String] = (1, "hello")
    println(s"tuple ${tuple._1}, ${tuple._2}")
  }
}

File: kotlin/src/main/kotlin/io/digitalstream/library/Tuples.kt
/**
 * Created by sebby on 9/25/16.
 */
package io.digitalstream.library

object Tuples {

  fun run(): Unit {
    val tuple = Pair(1, "hello")
    val tuple2: Pair<Int, String> = Pair(1, "hello")
    println("tuple ${tuple.first}, ${tuple.second}")
  }
}
